packages/i18n/src/cs.ts

```diff
@@ -0,0 +1,74 @@
+    isoDate:        (issue) => `Neplatné datum: Obdrženo ${issue.received}`,
+    isoDateTime:    (issue) => `Neplatné datum a čas: Obdrženo ${issue.received}`,
+    isoTime:        (issue) => `Neplatný čas: Očekáváno "hh:mm", ale obdrženo ${issue.received}`,
+    isoTimeSecond:  (issue) => `Neplatný čas: Očekáváno "hh:mm:ss", ale obdrženo ${issue.received}`,
+    isoTimestamp:   (issue) => `Neplatný časový údaj: Očekáván ISO 8601 formát, ale obdrženo ${issue.received}`,
+    isoWeek:        (issue) => `Neplatný týden: Obdrženo ${issue.received}`,
+    length:         (issue) => `Neplatná délka: Očekáváno ${issue.expected}, ale obdrženo ${issue.received}`,
+    mac:            (issue) => `Neplatná MAC adresa: Obdrženo ${issue.received}`,
+    mac48:          (issue) => `Neplatná 48-bitová MAC adresa: Obdrženo ${issue.received}`,
+    mac64:          (issue) => `Neplatná 64-bitová MAC adresa: Obdrženo ${issue.received}`,
+    maxBytes:       (issue) => `Překročena maximální hodnota v bajtech (bytes): Očekáváno ${issue.expected}, ale obdrženo ${issue.received}`,
+    maxLength:      (issue) => `Překročena maximální délka: Očekáváno ${issue.expected}, ale obdrženo ${issue.received}`,
+    maxSize:        (issue) => `Překročena maximální velikost: Očekáváno ${issue.expected}, ale obdrženo ${issue.received}`,
+    maxValue:       (issue) => `Překročena maximální hodnota: Očekáváno ${issue.expected}, ale obdrženo ${issue.received}`,
+    mimeType:       (issue) => `Neplatný MIME typ: Očekáváno ${issue.expected}, ale obdrženo ${issue.received}`,
+    minBytes:       (issue) => `Nedostatečná minimální délka bajtů (bytes): Očekáváno ${issue.expected}, ale obdrženo ${issue.received}`,
+    minLength:      (issue) => `Nedostatečná minimální délka: Očekáváno ${issue.expected}, ale obdrženo ${issue.received}`,
+    minSize:        (issue) => `Nedostatečná minimální velikost: Očekáváno ${issue.expected}, ale obdrženo ${issue.received}`,
+    minValue:       (issue) => `Nedostatečná minimální hodnota: Očekáváno ${issue.expected}, ale obdrženo ${issue.received}`,
+    multipleOf:     (issue) => `Neplatný násobek: Očekáváno ${issue.expected}, ale obdrženo ${issue.received}`,
+    nanoid:         (issue) => `Neplatné Nano ID: Obdrženo ${issue.received}`,
+    nonEmpty:       (issue) => `Nedostatečná délka: Očekáváno ${issue.expected}, ale obdrženo ${issue.received}`,
+    notBytes:       (issue) => `Nedostatečná minimální délka bajtů (bytes): Očekáváno ${issue.expected}, ale obdrženo ${issue.received}`,
+    notLength:      (issue) => `Nedostatečná délka: Očekáváno ${issue.expected}, ale obdrženo ${issue.received}`,
+    notSize:        (issue) => `Nedostatečná minimální velikost: Očekáváno ${issue.expected}, ale obdrženo ${issue.received}`,
+    notValue:       (issue) => `Nedostatečná minimální hodnota: Očekáváno ${issue.expected}, ale obdrženo ${issue.received}`,
+    octal:          (issue) => `Neplatné osmičkové číslo: Obdrženo ${issue.received}`,
```

Comment:
Does it make sense to use "osmičkové" when "hexadecimální" is used? Wouldn't "oktalové" make more sense?